core/lease/store.go

```diff
@@ -120,9 +120,17 @@ func ValidateString(s string) error {
 	return nil
 }
 
-// ErrInvalid indicates that a Store operation failed because latest state
-// indicates that it's a logical impossibility. It's a short-range signal to
-// calling code only; that code should never pass it on, but should inspect
-// the Store's updated Leases() and either attempt a new operation or return
-// a new error at a suitable level of abstraction.
-var ErrInvalid = errors.New("invalid lease operation")
+var (
+	// ErrInvalid indicates that a Store operation failed because latest state
+	// indicates that it's a logical impossibility. It's a short-range signal to
+	// calling code only; that code should never pass it on, but should inspect
+	// the Store's updated Leases() and either attempt a new operation or return
+	// a new error at a suitable level of abstraction.
+	ErrInvalid = errors.New("invalid lease operation")
```

Comment:
For future work perhaps, we have Invalid and now Timeout errors in the errors package.